README_KO.md

```
@@ -0,0 +1,196 @@
+
+| 파라미터 | 설명 | 기본값 |
+| :--- | :--- | :--- |
+| `state` | Picker를 제어하기 위한 상태 객체입니다. | `rememberYearMonthPickerState()` |
+| `startLocalDate` | Picker에 설정될 초기 날짜입니다. | `currentDate` |
+| `yearItems` | 선택 가능한 연도 목록입니다. | `1900..2100` |
+| `monthItems` | 선택 가능한 월 목록입니다. | `1..12` |
+| `visibleItemsCount` | 리스트에 표시될 아이템의 개수입니다. | `3` |
+
+## 라이선스
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)